ddtrace/tracer/payload.go

```diff
@@ -20,18 +20,6 @@ import (
 //
 // payload is not safe for concurrent use.
 //
-// Example:
-//
-//   p := newPayload()
-//   // add some items
-//   p.push(1)
-//   p.push(2)
-//   p.push(3)
-//   // decode into a slice
-//   var numbers []int
-//   codec.NewDecoder(p, &codec.MsgpackHandler{}).Decode(&numbers)
```

Comment:
I didn't make sense anymore for the new code, since you can't push any value other than a `[]*span` now. And writing an example with `[]*span` would have been too complicated.

Reply:
👍